Задание для кандидатов. Выполнить ревью.sql

```diff
@@ -0,0 +1,123 @@
+create procedure syn.usp_ImportFileCustomerSeasonal
+	@ID_Record int
+AS
+set nocount on
+begin
+	declare @RowCount int = (select count(*) from syn.SA_CustomerSeasonal)
+	declare @ErrorMessage varchar(max)
+
+-- Проверка на корректность загрузки

```

Comment:
Комментарий 3
комментирование идет на уровне строки или блока который комментируется
```suggestion
           -- Проверка на корректность загрузки

```